lib/spack/spack/ci.py

```diff
@@ -833,6 +1026,13 @@ def generate_gitlab_ci_yaml(
         else:
             broken_spec_urls = web_util.list_url(broken_specs_url)
 
+    # Make sure all reserved tags are removed for all jobs
```

Comment:
I don't think this comment makes sense here.

Reply:
I think my comment here is still valid, isn't it?

Reply:
yup, this got moved into `generate_ir`